lib/rpc/native-interaction/rubix_service.dart

```diff
@@ -174,4 +174,20 @@ class RubixService {
         getConnection(gateway: gateway, accessToken: accessToken);
     return stub.streamIncomingTxn(Empty());
   }
+
+  Future<Token> generateAccessToken(
+      {required String gateway,
+      required String accessToken,
+      required String privateKeyString}) async {
+    RubixServiceClient stub =
+        getConnection(gateway: gateway, accessToken: accessToken);
+    var challenge = await stub.getAccessTokenChallenge(Empty());
```

Comment:
name this variable as resp or response, so that re can read challengeString as resp.challenge rather than challenge.challenge?

Reply:
Will make the changes mentioned .